services/com.mde-network.ep.toolfunctions.xtext/src/main/java/com/mdenetnetwork/ep/toolfunctions/xtext/XtextTool.java

```diff
@@ -69,18 +69,32 @@ public void run(String languageName, String baseName, String extension, String g
 		
 		CliProjectsCreatorMain.main(args);
 		
-		
-		
-		
 		// Add activity files
-		if (grammar != null) {
-			final String grammarSrcPath = languageName.replace('.', '/');
-			final String xtextGrammarPath= PROJECT_PATH + baseName + "/src/" + grammarSrcPath  + ".xtext";
-			recreateGrammarFile(xtextGrammarPath, grammar);
+        //     TODO support multiple file types
```

Comment:
Can this comment go?

Reply:
Removed in 766e28f1cad63b46f7f0ed02028e89688b1e94c4